packages/mobile/src/navigator/NavigatorWrapper.tsx

```diff
@@ -132,6 +132,12 @@ export const NavigatorWrapper = () => {
     }
   }, [appState])
 
+  React.useEffect(() => {
```

Comment:
I think we can merge this into the `useEffect` on line 88 because that is also triggering on `componentDidMount`

Reply:
Spoke with Gonza a bit about this because I wasn't sure how the effects work but we agreed to keep the effect separate since it looks cleaner, and also we are returning a function to be used on unmount.